db/migrate/20230403140000_create_created_ids.rb

```diff
@@ -7,6 +7,8 @@ def up
       t.datetime :hour, null: false
       t.bigint :min_id, null: false
       t.bigint :max_id, null: false
+
+      t.timestamps
```

Comment:
Do you think these columns make sense? Or should we just disable the rubocop rule?

Reply:
We can add the rule since we already have a datetime